app/assets/stylesheets/overrides/breadcrumb.scss

```diff
@@ -2,3 +2,7 @@
   background-color: $light;
   font-size: 0.9rem;
 }
+
+.breadcrumb-item {
+  color: $breadcrumb-gray !important;
```

Comment:
Does it pass if we use `$dark-gray`? That's what we used for https://github.com/pulibrary/dpul/pull/1246/files and if we don't have to create a new color variable I think it would be nice.